release/pom.xml

```diff
@@ -31,6 +56,11 @@
           <name>jetty-download</name>
         </property>
       </activation>
+      <!--activation>
+        <file>
+            <missing>jetty</missing>
+        </file>
+      </activation-->
```

Comment:
Merged. It would be useful to create a new PR to activate the download of jetty automatically if the folder doesn't exist. I could avoid human mistakes during the release process, @jodygarnett.

Reply:
Yes, I think I can replace the build.xml script and do all of this with maven to make it easier to manage.

Thanks for review and merge, was going for minimal change, not downloading jetty avoids building installer each time if you are not interested in that.